### ISSUE TITLE
Investigate and fix bug with displaying dates so we can merge Service end date feature

### ISSUE THREAD

**Issue by alexsoble** [studentinsights/studentinsights#1101]
# Issue description 

https://github.com/studentinsights/studentinsights/pull/1100#issue-260753987

# Notes 

@jngai, I think the best way to debug this would be to log into the demo site (either locally or on the web), click through to a Student Profile, and explore why the dates are being rendered as "Invalid date."

**Comment by Jngai** [comment]
@alexsoble Quick question, that invalid date is due to a new field. Did you migrate the production/staging database?

**Comment by Jngai** [comment]
Also for existing services data in production, there may not be anything stored for `estimated_end_date` either.

**Comment by alexsoble** [comment]
@Jngai Good point. 

I'd suggest we add a migration to backfill all of that missing data when possible . . . if a future DiscontinuedService exists then it could be used as the estimated end date, for example.

And when there's no data we can use to backfill, I'd suggest we simply remove the "Scheduled at" line instead of having it say "Scheduled to End Invalid date." The latter isn't user-friendly, it's confusing.

**Comment by Jngai** [comment]
@alexsoble I added a valid check on the datetime in my new pr #1105 , plus a I wrote a rake task to backfill the services estimated end date with their discontinuedservice discontinued at. Since a service can have many discontinuedservice, I just chose the first one. If its the last one, let me know. I tried locally and is working.

Do we have an auto deploy thing going on so all master commits head into demo site?
